### DISCUSSION TITLE
Maintaining a custom Speech Responder Personality

### DISCUSSION THREAD

**Discussion by ACyprus** [EDCD/EDDI#2653] [EDDI Standalone Q&A] [answered]
Hey Team

I'm after some advice, please, on how best to maintain my custom Personality.

I'd essentially like to benefit from any new stuff added to the default Personality, whilst using my own custom version.

My concern is primarily FOMO, that my Personality will fall behind the default EDDI version when new features, key improvements or even additional scripts are made during an EDDI update. For example, I'm returning to the game after a couple years' break, and worry that my custom Personality will now be missing lots of Oddyssey related stuff or other nifty updates made during my absence.

Is there a more efficient approach beyond tracking notes in `ChangeLog.md` rather than diffing mine against the default `eddi.json` with each release, (or even making a copy of `eddi.json` and diffing that against the new version?

I'm aware of the super `Compare to default` feature when viewing each script individually, and that I can see which are different in the list by presence of an enabled `Reset` button, but I'm looking for a more all-in-one approach, I suppose.

Thanks for any help!

**Comment by Tkael** [comment, marked as answer]
The `Compare to default` button is probably still the best option if you want to diff against your own personality.
Another option would be to give up on maintaining your own custom personality if your activity in the game waxes and wanes. If you'd like something other than the default then quite a few people seem to be using [the personality that Darkcyde publishes and maintains against the latest EDDI releases](https://forums.frontier.co.uk/threads/eddi-scripts-and-eddi-enabled-va-commands-thread.311614/post-10394418).

**Comment by ACyprus** [comment]
Thanks, will probably continue with `Compare to default` or just diffing the lot manually. I prefer not to use Darkcyde's as then I'd lost my own personal tweaks :)
